clients/java/zts/core/src/test/java/com/yahoo/athenz/zts/ZTSClientTest.java

```diff
@@ -3014,6 +3019,31 @@ public void testGetAccessTokenCacheKeySSLContext() {
         client.close();
     }
 
+    @Test
+    public void testGetAccessTokenFromFile() {
```

Comment:
we should add test cases where we have invalid access tokens on disk and make sure those are correctly handled and skipped by the client

Reply:
Added invalid access token